docs/source/configuration.rst

```diff
@@ -151,25 +144,30 @@ Example owner & admin configurations::
 
     # Using nickname matching
     [core]
-    owner = dgw
+    owner = dgw        # used for alerts and ownership verification
```

Comment:
Since I've seen people on IRC using our example directly, they had an issue with such comments: the config parser thinks that anything after the `=` is part of the content, **including** what we *think* is a comment, but isn't for the config parser!

So yeah, this line, for the config parser, means this:

```
>>> bot.settings.core.owner
'dgw # used for alerts and ownership verification'
```

I think we should not show example with this syntax, as it will (and it already did) confuse some readers.

Reply:
Phooey. If this was for 8.0 I'd say let's tweak the parser behavior (it's totally possible to support this syntax), but the patch is destined for 7.1.x maintenance…

I'll have to make a pass through the docs converting this style to comments on their own line(s). Not a big deal to fold it into this PR, but frustrating.